packages/ethereum/package.json

```diff
@@ -7,6 +7,7 @@
   "homepage": "https://hoprnet.org",
   "main": "./lib/index.js",
   "scripts": {
+    "prepack": "rm -rf deployments/hardhat-localhost deployments/hardhat-localhost2",
```

Comment:
what is this used for?

Reply:
previously that deployment info would be published within the npm packages as well. To prevent that this hook is called by npm/yarn before packaging and will remove the deployment info we don't want in the packages.